3_Implementation/documentation/html/search/enumvalues_0.js

```diff
@@ -0,0 +1,36 @@
+var searchData=
+[
+  ['unity_5farray_5fto_5farray_911',['UNITY_ARRAY_TO_ARRAY',['../unity__internals_8h.html#a229eb7ffd452e7c158f1fe19005a817ca6e1eb5eefbb4a2bfed70942a3dc274eb',1,'unity_internals.h']]],
+  ['unity_5farray_5fto_5fval_912',['UNITY_ARRAY_TO_VAL',['../unity__internals_8h.html#a229eb7ffd452e7c158f1fe19005a817ca577dd656fdb92684b33920d5986d6f12',1,'unity_internals.h']]],
+  ['unity_5farray_5funknown_913',['UNITY_ARRAY_UNKNOWN',['../unity__internals_8h.html#a229eb7ffd452e7c158f1fe19005a817ca15a7d1868025012dc9f717205f9a3b3f',1,'unity_internals.h']]],
+  ['unity_5fdisplay_5fstyle_5fchar_914',['UNITY_DISPLAY_STYLE_CHAR',['../unity__internals_8h.html#a828517387e75bc0b3ee6a99d2e0722a4a21695b9f50194fbf29cd884cd0251415',1,'unity_internals.h']]],
+  ['unity_5fdisplay_5fstyle_5fhex16_915',['UNITY_DISPLAY_STYLE_HEX16',['../unity__internals_8h.html#a828517387e75bc0b3ee6a99d2e0722a4ad34011b6a340663cf55a595f87490462',1,'unity_internals.h']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["unity_5fdisplay_5fstyle_5fhex16_915",['UNITY_DISPLAY_STYLE_HEX16',['../unity__internals_8h.html#a828517387e75bc0b3ee6a99d2e0722a4ad34011b6a340663cf55a595f87490462',1,'unity_internals.h']]],
```